mycoffee/functions.py

```diff
@@ -94,7 +94,8 @@ def print_result(params: Dict[str, Union[str, int, float, dict]], ignore_warning
     if not ignore_warnings:
         warnings_list = params["warnings"]
         if len(warnings_list) > 0:
-            print("\n".join(warnings_list))
+            for message in warnings_list:
+                print("[Warning] " + message)
```

Comment:
Just a minor note on this. This will add a new line at the end. It's not a big deal, but I wanted to mention that if we want to say something about this change in the changelog.

Reply:
I believe it's not a significant issue, and it may also improve readability.